tests/queries/0_stateless/03460_projection_part_filtering_and_introspection.reference

```diff
@@ -0,0 +1,85 @@
+(
+    id UInt64,
+    event_date Date,
+    user_id UInt32,
+    url String,
+    region String,
+    PROJECTION region_proj
+    (
+        SELECT _part_offset ORDER BY region
+    ),
+    PROJECTION user_id_proj
+    (
+        SELECT _part_offset ORDER BY user_id
+    )
+)
+ENGINE = MergeTree
+ORDER BY (event_date, id)
+SETTINGS index_granularity = 1, max_bytes_to_merge_at_max_space_in_pool = 1; -- disable merge
+INSERT INTO test_simple_projection VALUES (1, '2023-01-01', 101, 'https://example.com/page1', 'europe');
+INSERT INTO test_simple_projection VALUES (2, '2023-01-01', 102, 'https://example.com/page2', 'us_west');
+INSERT INTO test_simple_projection VALUES (3, '2023-01-02', 106, 'https://example.com/page3', 'us_west');
+INSERT INTO test_simple_projection VALUES (4, '2023-01-02', 107, 'https://example.com/page4', 'us_west');
+INSERT INTO test_simple_projection VALUES (5, '2023-01-03', 104, 'https://example.com/page5', 'asia');
+SET enable_analyzer = 1;
+SET enable_parallel_replicas = 0;
+-- region projection is enough effective for filtering
+EXPLAIN projections = 1 SELECT * FROM test_simple_projection WHERE region = 'europe' AND user_id = 101;
+Expression ((Project names + Projection))
+  Expression
+    ReadFromMergeTree (default.test_simple_projection)
+    Projections:
+        Name: region_proj
+        Description: Projection has been analyzed and is used for part-level filtering
+        Condition: (region in [\'europe\', \'europe\'])
+        Parts: 1
+        Marks: 1
+        Ranges: 1
+        Rows: 1
+        Filtered parts: 4
+-- Only user_id projection is effective for filtering
+EXPLAIN projections = 1 SELECT * FROM test_simple_projection WHERE region != 'unknown' AND user_id = 106;
+Expression ((Project names + Projection))
+  Expression
+    ReadFromMergeTree (default.test_simple_projection)
+    Projections:
+        Name: user_id_proj
+        Description: Projection has been analyzed and is used for part-level filtering
+        Condition: (user_id in [106, 106])
+        Parts: 1
+        Marks: 1
+        Ranges: 1
+        Rows: 1
+        Filtered parts: 4
+-- Both region and user_id projections are effective for filtering
+EXPLAIN projections = 1 SELECT * FROM test_simple_projection WHERE region = 'us_west' AND user_id = 107;
+Expression ((Project names + Projection))
+  Expression
+    ReadFromMergeTree (default.test_simple_projection)
+    Projections:
+        Name: region_proj
+        Description: Projection has been analyzed and is used for part-level filtering
+        Condition: (region in [\'us_west\', \'us_west\'])
+        Parts: 3
+        Marks: 3
+        Ranges: 3
+        Rows: 3
+        Filtered parts: 2
+        Name: user_id_proj
+        Description: Projection has been analyzed and is used for part-level filtering
+        Condition: (user_id in [107, 107])
+        Parts: 1
+        Marks: 1
+        Ranges: 1
+        Rows: 1
+        Filtered parts: 2
```

Comment:
Let's make ident like
```
    Projections:
      Name: region_proj
        Description: Projection has been analyzed and is used for part-level filtering
        Condition: (region in [\'us_west\', \'us_west\'])
        Parts: 3
        Marks: 3
        Ranges: 3
        Rows: 3
        Filtered parts: 2
      Name: user_id_proj
        Description: Projection has been analyzed and is used for part-level filtering
        Condition: (user_id in [107, 107])
        Parts: 1
        Marks: 1
        Ranges: 1
        Rows: 1
        Filtered parts: 2
```
or similar, to identify where one projection starts and another one ends